##[pylyzer] failed /home/vizn3r/dev/dano-picovina/pdf/venv/lib/python3.12/site-packages/cryptography/exceptions.py 1748113092 1087
.___v_desugar_1: Never
.annotations: Never
.typing: Never

.___v_desugar_2: Never
.rust_exceptions: Never

.___v_desugar_3: Never
.rust_openssl: Never


._Reasons: Never
.UnsupportedAlgorithm: ClassType
.UnsupportedAlgorithm <: global::Exception
.UnsupportedAlgorithm._reason: Never
.UnsupportedAlgorithm.__call__: (message: Str, reason: Obj := Obj) -> cryptography.exceptions.UnsupportedAlgorithm

.AlreadyFinalized: ClassType
.AlreadyFinalized <: global::Exception

.AlreadyUpdated: ClassType
.AlreadyUpdated <: global::Exception

.NotYetFinalized: ClassType
.NotYetFinalized <: global::Exception

.InvalidTag: ClassType
.InvalidTag <: global::Exception

.InvalidSignature: ClassType
.InvalidSignature <: global::Exception

.InternalError: ClassType
.InternalError <: global::Exception
.InternalError.err_code: global::List!(Never, _: Nat)
.InternalError.__call__: Never

.InvalidKey: ClassType
.InvalidKey <: global::Exception

